src/kit/Progress.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import { floatToPercent } from 'kit/internal/string';
```

Comment:
nit: maybe just define it this in this file instead of importing?

Reply:
👍